packages/ui/src/components/file-control-bars/file-viewer-control-bar.tsx

```diff
@@ -43,7 +43,7 @@ export const FileViewerControlBar: FC<FileViewerControlBarProps> = ({
 
   const RightDetails = () => {
     return (
-      <ButtonGroup verticalAlign="center" spacing="2">
+      <ButtonLayout className="items-center gap-2">
```

Comment:
Kinda looks like it shouldn't have been a `ButtonGroup` in the first place. I'd consider taking `ButtonLayout` out of here and use a `Layout.Horizontal` instead.